src/platform/Darwin/LoggingImpl.cpp

```diff
@@ -0,0 +1,31 @@
+/*
+ *
+ *    Copyright (c) 2021 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <platform/logging/LogV.h>
+
+namespace chip {
+namespace Logging {
+namespace Platform {
+
+void LogV(const char * module, uint8_t category, const char * msg, va_list v)
+{
+    /* no-op, ChipPlatformLog expands to an os_log call directly (see Logging.h) */
```

Comment:
Hmm.  And nothing calls LogV directly?

Reply:
Our Darwin `ChipPlatformLog` still also calls `ChipInternalLogImpl`, which handles redirection and calling this `Platform::LogV` method. I was originally going to remove this, but having this call still in the build-time swapping in of the `stdio` backend keeps working. This was previously relied upon implicitly by some targets and I'm using it more explicitly now for chip-tool and all unit tests. I will add some more detail to the comment in the code.